src/applications/personalization/profile-2/components/MHVTermsAndConditionsStatus.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import recordEvent from 'platform/monitoring/record-event';
+
+import Verified from './Verified';
+
+const MHVTermsAndConditionsStatus = ({ mhvAccount }) => {
+  const termsAndConditionsUrl =
+    '/health-care/medical-information-terms-conditions';
+
+  if (mhvAccount.termsAndConditionsAccepted) {
+    return (
+      <Verified>
+        <>
+          You’ve accepted the latest Terms and Conditions for Medical
+          Information.
+          <p className="vads-u-margin-bottom--0">
+            <a
+              href={termsAndConditionsUrl}
+              onClick={() =>
+                recordEvent({
+                  event: 'account-navigation',
+                  'account-action': 'view-link',
+                  'account-section': 'terms',
+                })
+              }
+            >
+              View terms and conditions for medical information
+            </a>
+          </p>
+        </>
+      </Verified>
+    );
+  } else if (mhvAccount.accountState === 'needs_terms_acceptance') {
+    return (
+      <>
+        <p className="vads-u-margin--0">
+          To get started using our health tools, you’ll need to read and agree
+          to the Terms and Conditions for Medical Information. This will give us
+          your permission to show you your VA medical information on this site.
+        </p>
+        <p className="vads-u-margin-bottom--0">
+          <a
+            href={termsAndConditionsUrl}
+            onClick={() =>
+              recordEvent({
+                event: 'account-navigation',
+                'account-action': 'view-link',
+                'account-section': 'terms',
+              })
+            }
+          >
+            Go to the Terms and Conditions for Health Tools
+          </a>
+        </p>
+      </>
+    );
+  }
+  return null;
+};
+
+/**
+ * A static helper so that potential parent components will know if this
+ * component will render content at all and potentially adjust their own
+ * rendering logic to account for that. For example, if the parent component
+ * will wrap this component in other markup and it wants to avoid making that
+ * additional wrapper markup if this component will not render anything.
+ */
+MHVTermsAndConditionsStatus.willRenderContent = mhvAccount =>
```

Comment:
This is the only "weird" thing about this PR. I wanted a way for the parent `AccountSecurityContent` component will know if it needs to add this component to a table view or not. The parent could do this same check itself, but it felt like the logic to determine if this `MHVTermsAndConditionsStatus` component would render anything should live with the component itself. Is there a better way to do this? Could the parent do a sort of pre-render of this component and if it actually rendered anything then it could be added to the table view?

Reply:
Hm, I see what you mean. I suppose it could be exported as a standalone selector? Or if it returns `null` anyway from the `render` if does have the right conditions, maybe the table could just filter it?

Reply:
Hmm, the standalone selector sounds like the alternative I like best. I tried googling to see if my approach was something other people did or if it was some kind of anti-pattern or what. It feels like this is something React could support _somehow_. Simply duplicating logic that already exists in the render function just feels wrong :( I can't imagine I'm the only person who has run into need before.

Reply:
It's definitely a valid separation of concerns. I don't have any issue with it being this way really. I'd maybe bind it to class as a `static` function because the `static` keyword is fun.

Reply:
well this isn't a `class` so this is the best I can do :-D

Reply:
Nice that Sophie, who used to be on the core React team I believe, assures me that I'm not crazy in thinking there isn't really a great way to determine if a component will render https://stackoverflow.com/a/39060039/585275